### PR TITLE
[PBW-4368] fixes more options/control bar items separation logic

### PR DESCRIPTION
**Pull request by ksunika**

### CODE REVIEW

File: js/components/controlBar.js [hunk 1]
```diff
@@ -25,7 +26,7 @@ var ControlBar = React.createClass({
   componentWillReceiveProps: function(nextProps) {
     // if responsive breakpoint changes
     if (nextProps.responsiveView != this.props.responsiveView) {
-      this.responsiveUIMultiple = this.getResponsiveUIMultiple(this.props.responsiveView);
+      this.responsiveUIMultiple = this.getResponsiveUIMultiple(nextProps.responsiveView);
     }
```
Comment (ksunika) [inline js/components/controlBar.js]: this was causing incorrect responsive multiple when changing the screen size

File: scss/responsive/_xsmall.scss [hunk 2]
```diff
@@ -226,7 +226,7 @@
   .volumeBar {
     width: 6px * $xsmall-multiple;
     height: 18px * $xsmall-multiple;
-    padding-right: 3px * $xsmall-multiple;
+    padding-left: 3px * $xsmall-multiple;
   }
```
Comment (ksunika) [inline scss/responsive/_xsmall.scss]: this was causing volume bars to disappear on xsmall player

File: js/components/controlBar.js (hunk 1 above)
Comment (mistermendez) [inline js/components/controlBar.js]: good catch

**Comment by mistermendez** [comment]
sweet :candy:

**Pull request merged**